packages/qr-code/src/qr-code.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from "react";
+import QRCode from "qrcode.react";
+
+export interface QrProps {
+  value: string;
+  renderAs?: "canvas" | "svg" | undefined;
+  size: number;
+  style?: React.CSSProperties;
+  bgColor?: string | undefined;
+  fgColor?: string | undefined;
+  correctionLevel: "L" | "M" | "H";
+  includeMargin?: boolean | undefined;
+  iconSrc?: string | undefined;
+  iconSize?: number | undefined;
+  iconBg?: boolean | undefined;
+}
+
+interface IconSettings {
+  src: string;
+  x: number | undefined;
+  y: number | undefined;
+  height: number;
+  width: number;
+  excavate: boolean;
+}
+
+export const QrCode: React.FC<QrProps> = ({
```

Comment:
The use of React.FC in TypeScript with React is declining because it adds unnecessary explicitness, implicitly includes children, complicates default props, and is less aligned with future React patterns.
```suggestion
export const QrCode: QrProps = ({
```
What do you think?